index.bs

```diff
@@ -269,6 +269,49 @@ See also:
 
 * [[LEAST-POWER]]
 
+<h3 id="high-level-low-level">Resolving tension between high level and low level APIs</h3>
+
+<!--
+See "High level or low-level APIs?"
+https://github.com/w3ctag/design-principles/issues/117
+-->
+
+High-level APIs allow user agents more ability to
+intervene in various ways [on behalf of the user](#priority-of-constituencies),
+such as to ensure accessibility, privacy, or usability.
+
+<div class=example id=font-enumeration>
+
+A font picker (high level API)
+[was recommended by the TAG](https://github.com/w3ctag/design-reviews/issues/399)
+over a Font Enumeration API (low level API)
+as it addresses the bulk of use cases,
+without the fingerprinting concerns that accompany
```

Comment:
I like the example but your leading point to it was about how good (high level API) are as they provide accessibility, privacy etc. You should add a sentence about Font Picker being accessible, privacy preserving etc.

Reply:
There was already a mention of accessibility and privacy that right under your comment, but I reworded to emphasize. What do you think?